src/components/NavBar/NavBar.css

```diff
@@ -1,18 +1,14 @@
 .navbar {
-  border-radius: 0px;
-  border: none;
-  margin: 0px;
-}
-
-.navbar .container {
-  width: 86%;
+    border-radius: 0px;
+    border: none;
+    margin: 0px;
 }
 
 .modal-btn-ctn {
-  text-align: right;
-  padding-right: 16px;
+    text-align: right;
+    padding-right: 16px;
 }
 
 .modal-btn-ctn .btn {
-  margin-left: 10px;
+    margin-left: 10px;
 }
```

Comment:
@Spetastian @MartyTheeMartian Are we using 2 spaces for CSS, or 4?  I don't care either way but let's just be consistent.